apps/web/src/components/Feed/CuratedFeed.tsx

```diff
@@ -1,42 +1,42 @@
 import { useCallback, useMemo } from 'react';
 import { graphql, useFragment, usePaginationFragment } from 'react-relay';
 
-import { CuratedThenGlobalFeedFragment$key } from '~/generated/CuratedThenGlobalFeedFragment.graphql';
-import { CuratedThenGlobalFeedGlobalFragment$key } from '~/generated/CuratedThenGlobalFeedGlobalFragment.graphql';
-import { CuratedThenGlobalFeedGlobalPaginationQuery } from '~/generated/CuratedThenGlobalFeedGlobalPaginationQuery.graphql';
-import { CuratedThenGlobalFeedTrendingFragment$key } from '~/generated/CuratedThenGlobalFeedTrendingFragment.graphql';
-import { CuratedThenGlobalFeedTrendingPaginationQuery } from '~/generated/CuratedThenGlobalFeedTrendingPaginationQuery.graphql';
+import { CuratedFeedFragment$key } from '~/generated/CuratedFeedFragment.graphql';
+import { CuratedFeedGlobalFragment$key } from '~/generated/CuratedFeedGlobalFragment.graphql';
+import { CuratedFeedGlobalPaginationQuery } from '~/generated/CuratedFeedGlobalPaginationQuery.graphql';
+import { CuratedFeedPaginationFragment$key } from '~/generated/CuratedFeedPaginationFragment.graphql';
+import { CuratedFeedPaginationQuery } from '~/generated/CuratedFeedPaginationQuery.graphql';
 import isFeatureEnabled, { FeatureFlag } from '~/utils/graphql/isFeatureEnabled';
 
 import { useTrackLoadMoreFeedEvents } from './analytics';
 import { ITEMS_PER_PAGE } from './constants';
 import FeedList from './FeedList';
 
 type Props = {
-  queryRef: CuratedThenGlobalFeedFragment$key;
+  queryRef: CuratedFeedFragment$key;
 };
 
-export default function CuratedThenGlobalFeed({ queryRef }: Props) {
+export default function CuratedFeed({ queryRef }: Props) {
   const query = useFragment(
     graphql`
-      fragment CuratedThenGlobalFeedFragment on Query {
+      fragment CuratedFeedFragment on Query {
         ...FeedListFragment
 
-        ...CuratedThenGlobalFeedGlobalFragment
-        ...CuratedThenGlobalFeedTrendingFragment
+        ...CuratedFeedGlobalFragment
+        ...CuratedFeedPaginationFragment
         ...isFeatureEnabledFragment
       }
     `,
     queryRef
   );
 
   const globalPagination = usePaginationFragment<
```

Comment:
do we still need this?

Reply:
Removed it! My bad. I thought we have removed it from web 🤦‍♂️

Reply:
ohh all good, my bad I ONLY renamed it lol 😅